src/index.ts

```diff
@@ -1,7 +1,114 @@
+import sum from 'ml-array-sum';
+import variance from 'ml-array-variance';
+import covariance from 'ml-array-xy-covariance';
+import { Matrix, EigenvalueDecomposition } from 'ml-matrix';
+
 /**
- * My module
- * @returns A very important number
+ * Calculates the Directional Ellipse for a set of points with a specific standard deviation
+ * @param {object} [points]
```

Comment:
You should use the following type definition: https://github.com/cheminfo/cheminfo-types/blob/main/src/core/DataXY.d.ts